### PR TITLE
update nb to 5.7.2

### PR DESCRIPTION
**Pull request by eteq**
Currently github complains of a security vulnerability in versions of the notebook < 5.7.2 .  This PR just updates the various places we give an nb version to address that.

cc @adrn

### CODE REVIEW

**Comment by adrn** [comment]
Why is the circleCI build failing? It's complaining about astropy helpers but I don't know what is trying to import the helpers...I thought I removed all references to the helpers. Did you pull from upstream before making this branch?

**Comment by pllim** [comment]
I think you need to add `sphinx-astropy` to your dependency. I saw a similar error on RTD for my other projects after upgrading to helpers 3.1.

File: .rtd-environment.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ dependencies:
   - numpy=1.14
```
Comment (pllim) [inline .rtd-environment.yml]: The indentation here bothers me.

File: .rtd-environment.yml [hunk 2]
```diff
@@ -12,7 +12,7 @@ dependencies:
...
   - scipy=1.0 # needed for coordinates cross-matching
   - jupyter=1.0
-  - notebook=5.4
+  - notebook>=5.7.2
```
Comment (pllim) [inline .rtd-environment.yml]:
```suggestion
  - notebook>=5.7.2
  - sphinx-astropy
```

Comment (pllim) [reply .rtd-environment.yml]: p.s. Well, the failure is at CircleCI, so probably need to update the other files as well but you get the idea.

p.p.s. Albeit this being an educated guess, I could be wrong. Only one way to find out!

Comment (adrn) [reply .rtd-environment.yml]: This is RTD - we actually don't use RTD anymore, and sphinx-astropy is in the circleci config

**Comment by adrn** [comment]
I just checked - it looks like this branch is based on an old version of master.

File: .rtd-environment.yml (hunk 2 above)
Comment (pllim) [reply .rtd-environment.yml]: Well... Ignore me then... :sweat_smile:

**Comment by adrn** [comment]
I rebased and now circleci is passing.

**Pull request merged**